examples/basic/README.md

```diff
@@ -0,0 +1,43 @@
+# Basic Example
+
+This example show an integration with Github repository webhooks.
+
+## Prerequisites
+
+- Install [ngrok](https://ngrok.com/)
+- Generate a github personal access token with scopes: ``
+- A github repository that you may create webhooks on.
+
+## Running
+
+- Start an ngrok tunnel to expose your server on port 3000
+  - `ngrok http :3000`
+- Set the following environment variables:
+  ```
+  PORT=3000
+  GITHUB_ACCESS_TOKEN=<YOUR PAT WITH REPOSITORY WEBHOOK SCOPES>
+  GITHUB_USERNAME=<USERNAME>
+  GITHUB_REPO_NAME=<REPO NAME>
+  NGROK_HOST=<NGROK TUNNEL HOST>
+  ```
+- start the fastify server: `node index.js`
+- create a webhook by making a POST request to /create-github-webhook
```

Comment:
I would say rather than have an endpoint in the example for creating the webhook, I would direct the user of the example to create a webhook themselves in github e.g. through the GUI (https://docs.github.com/en/developers/webhooks-and-events/webhooks/creating-webhooks#setting-up-a-webhook). My rationale is that it is clearer then what this module does and doesn't do (e.g. it isn't a module for creating webhooks on github). It also makes the example code itself simpler & clearer to see what is going on.